xml/System.Runtime.InteropServices/CoClassAttribute.xml

```diff
@@ -42,7 +42,7 @@
   
  When Tlbimp.exe imports a coclass, it produces a managed class and an interface to represent the coclass. The coclass interface has the same interface identifier (IID) as the default interface of the original coclass. The imported coclass interface also retains the name of the coclass. Tlbimp.exe appends the original coclass name with "class" to identify the imported class.  
   
- You rarely apply this attribute. However, if plan to write source code that produces metadata that closely simulates metadata produced by Tlbimp.exe, you should create a coclass interface for each coclass. Use the name of the original coclass to name the coclass interface and derive it from the default interface. In addition to the <xref:System.Runtime.InteropServices.CoClassAttribute>, you must also apply the <xref:System.Runtime.InteropServices.ComImportAttribute?displayProperty=nameWithType> and <xref:System.Runtime.InteropServices.GuidAttribute?displayProperty=nameWithType> to the coclass interface. For additional information about how Tlbimp.exe imports interfaces and coclasses from a type library, see [Imported Type Conversion](http://msdn.microsoft.com/library/cb1ee67d-8ef0-4e40-b2a5-13714b9f33c3).  
+ You rarely apply this attribute. However, if you plan to write source code that produces metadata that closely simulates metadata produced by Tlbimp.exe, you should create a coclass interface for each coclass. Use the name of the original coclass to name the coclass interface and derive it from the default interface. In addition to the <xref:System.Runtime.InteropServices.CoClassAttribute>, you must also apply the <xref:System.Runtime.InteropServices.ComImportAttribute?displayProperty=nameWithType> and <xref:System.Runtime.InteropServices.GuidAttribute?displayProperty=nameWithType> attributes to the coclass interface. For additional information about how Tlbimp.exe imports interfaces and coclasses from a type library, see [Imported Type Conversion](https://msdn.microsoft.com/library/cb1ee67d-8ef0-4e40-b2a5-13714b9f33c3(v=vs.100)) in the MSDN Library.  
   
```

Comment:
Great, this link is now functional.  Are you concerned that it is part of the 'archived' and 'not maintained' set of documents?